stablehlo/reference/Ops.cpp

```diff
@@ -817,18 +818,18 @@ SmallVector<InterpreterValue> eval(
       auto operand = scope.find(whileOp.getOperand());
       auto &cond = whileOp.getCond();
       auto &body = whileOp.getBody();
-      auto results = evalWhileOp(operand, cond, body, process, scope, fallback);
+      auto results = evalWhileOp(operand, cond, body, process, scope, config);
```

Comment:
Nit, lets be uniform with where the config arg appears. In `eval` it is before process/scope, in `evalWhileOp` it is a trailing argument. I don't have a strong preference on which position it should be, just that it is consistent.

Reply:
Decided to keep it before `Process` as I think once #1820 is addressed, we can make it a `const InterpreterConfiguration&` and keep it before any other optional pointer parameters.